app/settings.py

```diff
@@ -0,0 +1,126 @@
+"""
+Django settings for app project.
+
+Generated by 'django-admin startproject' using Django 5.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-hoyb4m59=e3&-$a))gj)2xg72!f_8h7q^40%976)hpm59&&yr!'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
The `DEBUG` setting is currently set to `True`. This is suitable for development but should be set to `False` in a production environment to prevent the display of sensitive information.